lib/Sema/CSSimplify.cpp

```diff
@@ -12898,6 +12907,7 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
   case FixKind::SpecifyLabelToAssociateTrailingClosure:
   case FixKind::AllowKeyPathWithoutComponents:
   case FixKind::IgnoreInvalidResultBuilderBody:
+  case FixKind::IgnoreInvalidASTNode:
```

Comment:
The impact of this fix should be `10` to underscore how broken the expression is.